pandas/core/frame.py

```diff
@@ -8124,7 +8124,7 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
 
         return self._constructor(correl, index=idx, columns=cols)
 
-    def cov(self, min_periods=None) -> "DataFrame":
+    def cov(self, min_periods=None, ddof=1) -> "DataFrame":
```

Comment:
Maybe add type annotations to these arguments (similar to nancov)?

Reply:
can you add types here